test/views/agent/tasklist/taxyear/WhatYearToSignUpViewSpec.scala

```diff
@@ -181,25 +182,31 @@ class WhatYearToSignUpViewSpec extends ViewSpec {
         form.attr("action") mustBe testCall.url
       }
 
-      "has a small size legend for radio options heading" in {
-        document().selectHead(".govuk-fieldset__legend").text() mustBe WhatYearToSignUp.radioSectionHeading
-
-      }
-
-      "has a current tax year radio button" in {
-        val radio: Element = document().select(".govuk-radios__item").get(0)
-        radio.select("input[id=accountingYear]").`val` mustBe "CurrentYear"
-        radio.select("label[for=accountingYear]").text mustBe Seq(
-          WhatYearToSignUp.currentYearOption((taxYearEnd - 1).toString, taxYearEnd.toString)
-        ).mkString(" ")
-      }
-
-      "has a next tax year radio button" in {
-        val radio: Element = document().select(".govuk-radios__item").get(1)
-        radio.select("input[id=accountingYear-2]").`val` mustBe "NextYear"
-        radio.select("label[for=accountingYear-2]").text mustBe Seq(
-          WhatYearToSignUp.nextYearOption(taxYearEnd.toString, (taxYearEnd + 1).toString)
-        ).mkString(" ")
+      "has the correct radio inputs" in {
+        document().mustHaveRadioInput(selector = "fieldset")(
+          name = "accountingYear",
+          legend = WhatYearToSignUp.radioSectionHeading,
+          isHeading = false,
+          isLegendHidden = false,
+          hint = None,
+          errorMessage = None,
+          radioContents = Seq(
+            RadioItem(
+              content = Text(Seq(
+                WhatYearToSignUp.currentYearOption((taxYearEnd - 1).toString, taxYearEnd.toString)
+              ).mkString(" ")),
+              value = Some("CurrentYear"),
```

Comment:
Should be able to replace this with a reference instead of a hard-coded string.
```suggestion
              value = Some(Current.toString),
```
Or something like that.
Same for the others